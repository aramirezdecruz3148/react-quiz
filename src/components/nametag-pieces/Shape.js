import React from 'react';
import PropTypes from 'prop-types';

function Shape({ text, textColor, backgroundColor }) {
  return (
    <div style={{ background: backgroundColor }}>
      <h3 style={{ color: textColor }}>{text}</h3>
    </div>
  );
}

Shape.propTypes = {
  text: PropTypes.string.isRequired,
  textColor: PropTypes.string.isRequired,
  backgroundColor: PropTypes.string.isRequired
};

export default Shape;
