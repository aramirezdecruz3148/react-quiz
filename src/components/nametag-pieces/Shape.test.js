import React from 'react';
import { shallow } from 'enzyme';
import Shape from './Shape';

describe('Shape component', () => {
  it('renders Shape', () => {
    const wrapper = shallow(<Shape text='testing' backgroundColor='white' textColor='purple' />);
    expect(wrapper).toMatchSnapshot();
  });
});
