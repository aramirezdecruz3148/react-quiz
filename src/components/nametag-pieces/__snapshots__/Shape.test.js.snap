// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shape component renders Shape 1`] = `
<div
  style={
    Object {
      "background": "white",
    }
  }
>
  <h3
    style={
      Object {
        "color": "purple",
      }
    }
  >
    testing
  </h3>
</div>
`;
